enclave/src/polkadex_balance_storage.rs

```diff
@@ -97,6 +107,27 @@ impl PolkadexBalanceStorage {
         }
     }
 
+    pub fn move_unconfirmed_balance(
```

Comment:
should this function not rather be called `withdraw_unconfirmed_balance` or `withdraw_confirmed`? A movement is something that is moved from somewhere to somewhere but in this function I can not see this.